detections/kerberoasting_spn_rquest_with_rc4_encryption.yml

```diff
@@ -0,0 +1,27 @@
+name: Kerberoasting spn request with RC4 encryption 
+id: 5cc67381-44fa-4111-8a37-7a230943f027
+version: 1
+date: '2020-06-11'
+description: This search detects a potential kerberoasting attack via service principal name requests
+how_to_implement: You must be ingesting endpoint data that tracks process activity, and include the windows security event logs that contain kerberos
+type: ESCU
+references:
+  - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1208/T1208.md
+  - https://www.trimarcsecurity.com/post/trimarcresearch-detecting-kerberoasting-activity
+author: Jose Hernandez, Splunk
+search: '`wineventlog_security` EventID=4769 TicketOptions=0x40810000 TicketEncryptionType=0x17 | `kerberoasting_spn_request_with_rc4_encryption_filter` '
```

Comment:
amigo, mind adding a table command or stats at the end of the search?

Reply:
and a macro file?

Reply:
no need for macro since its auto generated, fixed the table commands